core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -1047,7 +1047,20 @@ var ExplorationEditorMainTab = function () {
       'Post-tutorial popover takes too long to disappear'
     );
     await action.waitForAutosave();
+    await waitFor.elementToBeClickable(
+      stateNameContainer,
+      'State name container is taking too long to become clickable.'
+    );
+    // We need to use browser.pause() in order to wait for the state name
```

Comment:
I don't think you should have this? We try to avoid browser.pause in tests.

Reply:
Was left over from the commit I pushed to test something. Removed it now